rabbitmq-operator-model/src/main/java/com/indeed/operators/rabbitmq/model/crd/rabbitmq/AddressAndVhost.java

```diff
@@ -18,6 +20,9 @@ public AddressAndVhost(
             @JsonProperty("address") final String address,
             @JsonProperty("vhost") final String vhost
     ) {
+        Preconditions.checkArgument(!Strings.isNullOrEmpty(address), "'address' cannot be null");
```

Comment:
Nit: "cannot be null or empty" or words to that effect, to more closely resemble the actual test?